src/InputNumber/InputNumber.tsx

```diff
@@ -275,7 +275,7 @@ const InputNumber = React.forwardRef((props: InputNumberProps, ref) => {
     <Input
       {...(htmlInputProps as Omit<React.InputHTMLAttributes<HTMLInputElement>, 'size'>)}
       type="number"
-      autoComplete="off"
+      autoComplete={htmlInputProps?.pattern ? "hidden" : "off"}
```

Comment:
The code here doesn't solve entering a decimal point.  Also you need to format the code.  

You can run the command to check the code.
```
npm run lint   // or npm run test
```

If you just want the InputNumber to not be able to enter decimals, you can format the value in onChange .

```tsx
const App = () => {
  const [value, setValue] = React.useState(1);

  return (
    <InputNumber
      value={value}
      onChange={value => {
        setValue(parseInt(value));
      }}
    />
  );
};
```

Reply:
hi Simon,  I just linted the code and pushed.

Reply:
> The code here doesn't solve entering a decimal point.

My bad you're right. `pattern` attribute does not seem to work on input type=number. The code I wrote was probably browser-specific.

Will close the PR then.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#pattern